package/suse-migration-sle15-activation-spec-template

```diff
@@ -50,17 +45,11 @@ next reboot of the machine.
 python3 setup.py build
 
 %install
-python3 setup.py install --prefix=%{_prefix} --root=%{buildroot}
 
 install -D -m 755 grub.d/99_migration \
     %{buildroot}/etc/grub.d/99_migration
 
-mv %{buildroot}/%{_bindir}/suse-migration-pre-checks /tmp/
-rm %{buildroot}/%{_bindir}/*
-mv /tmp/suse-migration-pre-checks %{buildroot}/%{_bindir}/
-
 %post
-suse-migration-pre-checks
```

Comment:
WE still want to run the script, but should have the fully qualified path

Reply:
@rjschwei currently I only had modified the image.spec.in  in the IBS package, suse-migration-rpm to run this as a %post. I can add it here. as well.  The only downside is if a customer just  runs `zypper in suse-migration-sle15-activation (allowing zypper to pull in suse-migration-pre-checks and SLE15-Migration) , it will get run twice, once when SLES15-Migration is installed and once when suse-migration-sle15-activation in installed. Fine by me but wanted to make sure it as noted.  Let me know if this changes your mind at all.

Reply:
This is where I got confused, sorry. `-activation` shouldn't have a requires on `pre-checks` as you state `pre-checks` are run when `SLES15-Migration` is installed and one cannot install `-activation` without pulling `SLES15-Migration`. Wel, one can deliberately break things but that is a different conversation.

Reply:
ok So I just removed the requires. Please re-review